force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -44,35 +51,64 @@ public with sharing class VariablesDataTypesOperators {
      * @return The difference between the two numbers.
      */
     public static Integer subtraction(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+            Integer sub = a - b;
+        
+        return sub; // Replace null with the variable you used to store the result
+    }
+
+
+    public static void Results2(){
+        Integer callingMethod2 = subtraction(1,2);
+        System.debug('--Results--' + callingMethod2 );
     }
 
     /**
+     * Question 3
      * Performs multiplication of two numbers.
      * Example: multiplication(2, 3) should return 6
      * @param a The first number.
      * @param b The second number.
      * @return The product of the two numbers.
      */
     public static Integer multiplication(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+          
+           Integer sum = a * b;
+       
+        return sum; // Replace null with the variable you used to store the result
     }
+     
 
+    public static void Results3(){
+        Integer callingMethod3 = multiplication(2, 3);
+        System.debug('--Result--' + callingMethod3);
+    }
     /**
-     * Question 3
+     * Question 4
      * Performs division of two numbers.
      * If the denominator is zero, return 0 to avoid division by zero error.
      * Example: division(6, 3) should return 2
      * @param a The numerator.
      * @param b The denominator.
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
-    public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+    public static Decimal division(Decimal a, Decimal b) {
+            Decimal div = a / b;
+
+            return div;
+        
+         // Replace null with the variable you used to store the result
     }
 
+   public static void Results4(){
+        
+        Decimal callingMethod4 = division(6,3);
+        System.debug('Result = ' + callingMethod4);
+    }
```

Comment:
I would recommend to get in the habit of writing your own test class rather than calling it like this